src/acrcssc/setup.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+VERSION = '1.0.0b1'
```

Comment:
Why the version in history note is `1.1.1` but the version here is `1.0.0b1`?

Reply:
we changed to `1.0.0b1` to follow CLI versioning, but history didn't get updated. Will keep version `1.0.0b1` for this one and modify the HISTORY file